### PR TITLE
updated readme for new laravel polymorphic snippets

### PR DESCRIPTION
**Pull request by chinmaypurav**
Hi @mhmdomer, I missed out the docs in the previous PR.

Please have a look.

### CODE REVIEW

**Comment by chinmaypurav** [comment]
Also @mhmdomer , Did you release a new version after accepting PR2?

**Comment by mhmdomer** [comment]
> Also @mhmdomer , Did you release a new version after accepting PR2?

No, I didn't do another release yet, but I am planning to do it after merging this PR.

**Comment by mhmdomer** [comment]
@chinmaypurav The README.md is listing 8 snippets but there is actually 11 snippets. could you please add the other snippets to the README.md file?

File: README.md [hunk 1]
```diff
@@ -24,6 +26,11 @@ This snippet extension provides you with quick snippets for generating laravel r
 | belongsToMany  | Generates a belongsToMany relationship  |
 | hasManyThrough | Generates a hasManyThrough relationship |
 | hasOneThrough  | Generates a hasOneThrough relationship  |
+|    morphOne    | Generates a morphOne relationship       |
+|    morphTo     | Generates a morphTo relationship        |
+|    morphMany   | Generates a morphMany relationship      |
+|   morphToMany  | Generates a morphToMany relationship    |
+|  morphedByMany | Generates a morphedByMany relationship  |
```
Comment (mhmdomer) [inline README.md:33]: Please add the other snippets as well.

**Review by mhmdomer — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (chinmaypurav) [reply README.md:33]: @mhmdomer can you check it once? It is 11, You must have not expanded the file while looking at the changes.

**Pull request merged**